scripts/update-prefix-data.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Update the files in `src/prefix-data` to match the `browserList` below. Run
+ * this if browser support changes or `inline-style-prefixer` gets fixes for the
+ * supported browsers.
+ */
+import path from 'path'
```

Comment:
We need to lint the scripts.

Reply:
Done. I changed ESLint to lint everything so this forgotten-directories thing stops happening (turns out `test` was also not linted).